### PR TITLE
Wrappers for running Eppsilon on Brown's Oscar cluster

### PR DESCRIPTION
**Pull request by aelanman**
Long overdue, these are the wrappers I use for running eppsilon on Oscar.

### CODE REVIEW

**Comment by bhazelton** [comment]
Thanks @aelanman! It looks like this needs to be rebased, and I'm curious about the changes in the non-wrapper files (maybe those will go away after the rebase?)

I'm also wondering if these wrappers should live in the new pipeline_scripts repo or in eppsilon.

**Comment by aelanman** [comment]
I did do a rebase, but it's possible I forgot to pull the latest version of master before rebasing... let me try again.

**Comment by aelanman** [comment]
Having confirmed what was causing my awful power spectra earlier this week and fixed it in FHD, I dared to try rebasing again. I've run it and confirmed that eppsilon is working just fine now for me.

**Comment by bhazelton** [comment]
@aelanman sorry, I didn't mean to let this stagnate. Can you check out what's going on with the changes to the non-wrapper files? I'm not sure if you mean to make these changes.

**Comment by aelanman** [comment]
@bhazelton I'm not sure what the changes to `kpower_1d_plots` were for... but it seems to run just fine without those lines. This is freshly rebased. I'm pretty sure I've reverted most of the changes to non-wrapper files.

**Comment by bhazelton** [comment]
@aelanman I'm sorry, but I pushed a bug fix and now this needs to be rebased again so it doesn't undo the fix. I can do the rebasing if you want me to.

**Comment by aelanman** [comment]
No problem. Just rebased and pushed again.

File: ps_wrappers/ps_wrapper.pro [hunk 1]
```diff
@@ -46,7 +46,7 @@ pro ps_wrapper, folder_name_in, obs_name, data_subdirs=data_subdirs, $
     savefilebase = savefilebase, plot_path = plot_path, plot_filebase = plot_filebase, $
     individual_plots = individual_plots, plot_binning_hist = plot_binning_hist, $
     note = note, png = png, eps = eps, pdf = pdf, cube_power_info = cube_power_info, $
-    no_dft_progress = no_dft_progress, loc_name = loc_name
+    no_dft_progress = no_dft_progress, loc_name = loc_name, _Extra=extra
```
Comment (bhazelton) [inline ps_wrappers/ps_wrapper.pro]: Why are you adding this _Extra keyword here?
